mne_g3d/viz.py

```diff
@@ -65,27 +73,68 @@ def plot_brain_mesh(rh_vertices=None,
 
     fig = ipv.figure(width=fig_size[0], height=fig_size[1], lighting=True)
 
+    if act_data is not None:
+        cmap = get_mesh_cmap(act_data, cmap_str=cmap_str)
+        # data mapping into [0, 1] interval
+        arr_act_data = np.concatenate(tuple(act_data.values()))
+        dt_min = arr_act_data.min()
+        dt_max = arr_act_data.max()
+
+        out_min = 0
+        out_max = 1
```

Comment:
instead of doing this, put these values in the code below (it will simplify them) and add a comment like:
```
# normalize activity to the range [0, 1]
```